content/en/blog/releases/release_v0.6.0.md

```diff
@@ -0,0 +1,46 @@
+
+![openfunction-tracing-skywalking](/images/docs/en/blogs/openfunction-tracing-skywalking.jpg)
+
+Going forward, OpenFunction will add more capabilities for serverless functions in logging, metrics, and tracing. You will be able to use Apache SkyWalking and OpenFunction to set up a full-stack APM for your serverless workloads out-of-the-box. Moreover, OpenFunction will support OpenTelemetry that allows you to leverage Jaeger or Zipkin as an alternative.
+
+### Support Dapr pub/sub and bindings
+
+Dapr bindings allows you to trigger your applications or services with events coming in from external systems, or interface with external systems. OpenFunction 0.6.0 adds Dapr output bindings to its synchronous functions which enables HTTP triggers for asynchronous functions. For example, synchronous functions backed by Knative runtime can now interact with middleware via Dapr components acting like the Async functions. See this [guide](https://github.com/OpenFunction/samples/tree/main/functions/knative/with-output-binding) for the quickstart sample.
+
+​​OpenFuncAsync functions introduces [Dapr pub/sub](https://docs.dapr.io/developing-applications/building-blocks/pubsub/pubsub-overview/) to provide a platform-agnostic API to send and receive messages. A typical use case is that you can leverage synchronous functions to receive an event in plain JSON or Cloud Events format, and then send the received event to a Dapr output binding or pub/sub component, most likely a message queue (e.g. Kafka, NATS Streaming, GCP PubSub). Finally, the OpenFuncAsync function could be triggered from the message queue. See this [guide](https://github.com/OpenFunction/samples/tree/main/functions/async/pubsub) for the quickstart sample.
+
+![http-trigger-openfunction](/images/docs/en/blogs/http-trigger-openfunction.jpg)
+
+### Control autoscaling behavior for functions
+
+OpenFunction 0.6.0 integrates KEDA [ScaledObject](https://keda.sh/docs/2.5/concepts/scaling-deployments/#scaledobject-spec) spec which is used to define how KEDA should scale your application and what the triggers are. You just need to define the lower and upper bound in the function CRD without changing your code. 
+
+Meanwhile, it also provides the ability to control the concurrency and the number of simultaneous requests, which inherits the definition from [Dapr](https://docs.dapr.io/operations/configuration/control-concurrency/) and [Knative](https://knative.dev/docs/serving/autoscaling/concurrency/). A typical use case in distributed computing is to only allow for a given number of requests to execute concurrently. You are able to control how many requests and events will invoke your application simultaneously.
```

Comment:
The following could be removed because that's still something to do in the following releases and not done yet.

```
Meanwhile, it also provides the ability to control the concurrency and the number of simultaneous requests, which inherits the definition from [Dapr](https://docs.dapr.io/operations/configuration/control-concurrency/) and [Knative](https://knative.dev/docs/serving/autoscaling/concurrency/). A typical use case in distributed computing is to only allow for a given number of requests to execute concurrently. You are able to control how many requests and events will invoke your application simultaneously.
```

Reply:
Okay, just noticed that this feature is [still in progress](https://github.com/OpenFunction/OpenFunction/issues/165).

Reply:
@benjaminhuo Updated. I chose to preserve this paragraph to let more people keep an eye on it.